glossary.es.Rmd

```diff
@@ -1,3 +1,10 @@
-# Glosario 
+# Glosario
 
-## WIP
+## Fuente
```

Comment:
it might seem a bit useless to add headings but we need them at the moment, because if there's no TOC, no code links are added, and we append the link to the other language to the code links :sweat_smile:

Reply:
Ok! Looks great.  We can add some description about the Glossary and mention other sources we use as starting points.

Reply:
Do you want to modify my PR or should I merge it and you'd do a new PR?

Reply:
I will work on your PR (after our meeting ;-))

Reply:
Ok, thank you!